qutebrowser/components/misccommands.py

```diff
@@ -235,23 +236,39 @@ def click_element(tab: apitypes.Tab, filter_: str, value: str, *,
         value: The value to filter for.
         target: How to open the clicked element (normal/tab/tab-bg/window).
         force_event: Force generating a fake click event.
+        select_first: Select first matching element if there are multiple.
```

Comment:
I wonder if it should be an error if `--select-first` is specified with a filter which will only result in one element (like `id`)? On the other hand, we do indeed end up selecting the "first" (only) element, so it might be fine. What do you think?

Reply:
What does `:click-element id foo` do if there are more than one element with id foo? I know that this should not be the case but there are more than enough websites out there where it is. If this throws an error, than I would forbid `--select-first` for the other filters, since it would be unintuitive behavior. If this just clicks the first matching element, I would not bother with differentiating the `--select-first` behavior.

Reply:
From your link it seems to be unspecified:

> Since element IDs are required to be unique if specified, they're a useful way to get access to a specific element quickly.

Testing it seems to reliably return the first one, so I would keep the `--select-first` for all, as explained above.